docs/en/seatunnel-engine/download-seatunnel.md

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
What's different with https://github.com/apache/seatunnel/blob/dev/docs/en/start-v2/locally/deployment.md?

Reply:
In `[docs/en/seatunnel-engine/download-seatunnel.md](https://github.com/apache/seatunnel/pull/6947/files/7b6882f577a80a72590aa6f20c39585d9dec261e#diff-7e3b1ca5929ec947a03273a3cfa257ffdec2408332247c85b191fbe86e774de8)` the user only can see `SeaTunnel Zeta` link and do not show the `Flink` and `Spark`.